website/source/api/secret/ad/index.html.md

```diff
@@ -46,6 +46,14 @@ text that fulfills those requirements. `{{PASSWORD}}` must appear exactly once a
 * `userdn` (string, optional) - Base DN under which to perform user search. Example: `ou=Users,dc=example,dc=com`
 * `upndomain` (string, optional) - userPrincipalDomain used to construct the UPN string for the authenticating user. The constructed UPN will appear as `[username]@UPNDomain`. Example: `example.com`, which will cause vault to bind as `username@example.com`.
 
+### Other parameters
+
+* `last_rotation_tolerance` (string, optional) - Active Directory often shows a "pwdLastSet" time after Vault's because it takes
```

Comment:
Probably good to start with a basic definition before the explanation:

...(string, optional) - Tolerance duration to use when checking the last rotation time. Active Directory often...

Reply:
Ah good idea, will do.